tests/app.test.js

```diff
@@ -375,6 +376,15 @@ describe('InVEST subprocess testing', () => {
     module: 'natcap.invest.dot',
   };
 
+  const uiSpec = {
+    order: [['workspace_dir', 'results_suffix']],
+    argsOptions: {}
+  }
+  const uiSpecFilePath = path.join(
+    fileRegistry.INVEST_UI_DATA, `${spec.module}.json`
+  );
+  fs.writeFileSync(uiSpecFilePath, JSON.stringify(uiSpec));
+
```

Comment:
Will these files get cleaned up after the tests? We might want a `beforeAll` and `afterAll` functions to take care of that.

Reply:
Moved the `writeFileSync` into `beforeAll` and added `unlinkSync` to `afterAll`